code/game/machinery/portable_turret.dm

```diff
@@ -608,7 +617,8 @@
 	if(controllock)
 		return
 	src.enabled = TC.enabled
-	src.lethal = TC.lethal
+	if(egun) //If turret can switch modes.

```

Comment:
This needs to cover `src.lethal_icon = TC.lethal` too!

Reply:
Done